pie/default_optuna.json

```diff
@@ -0,0 +1,39 @@
+{
+      "params": [
+          // Example for changing a category
+          // {
+          //    "path": "cemb_type",
+          //    "type": "suggest_categorical",
+          //    "args": [["cnn", "rnn"]]
+          // }
+
+         // Example for changing an INT dimension
+         // {
+         //     "path": "cemb_dim",
+         //     "type": "suggest_int",
+         //     "args": [200, 600]
```

Comment:
Doesn't optuna allow to define sampling distributions for the parameters?

Reply:
The way I designed it, you can basically use any optuna distribution function. It just makes it simpler as you don't have to "code" it.
I have not yet implemented the GridSampler and stuff like this. Should probably be the next target... https://optuna.readthedocs.io/en/stable/reference/samplers.html?highlight=suggest#optuna.samplers.GridSampler

Reply:
Done ;)